test/SILGen/properties.swift

```diff
@@ -536,7 +564,7 @@ struct DidSetWillSetTests: ForceAccessors {
       // CHECK-NEXT: [[TAKEINTFN:%.*]] = function_ref @$S10properties7takeInt{{[_0-9a-zA-Z]*}}F
       // CHECK-NEXT: apply [[TAKEINTFN]]([[A]]) : $@convention(thin) (Int) -> ()
 
-      a = zero  // reassign, but don't infinite loop.
+      (self).a = zero  // reassign, but don't infinite loop.
```

Comment:
Why change the test?

Reply:
@jrose-apple To ensure we also look through paren exprs to the self base. We still check for a direct access when doing `a = zero` in `willSet`, should I swap them?

Reply:
Ah, I usually just prefer that new things be added rather than changing old things, but I forgot that the original form was tested above. It's fine to leave it, thanks.